src/dispatcher.rs

```diff
@@ -72,14 +84,54 @@ impl DBExecutor {
     }
 }
 
-impl Handler<CollectionInfo> for DBExecutor {
+impl Handler<Collections> for DBExecutor {
```

Comment:
Changed the name of this to better match the endpoint name and be more consistent with the others, hope that's cool.